frame/authorship/src/lib.rs

```diff
@@ -21,17 +21,36 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use codec::{Decode, Encode};
+use codec::{Decode, Encode, MaxEncodedLen};
 use frame_support::{
 	dispatch,
 	traits::{FindAuthor, Get, VerifySeal},
+	BoundedSlice, BoundedVec,
 };
 use sp_authorship::{InherentError, UnclesInherentData, INHERENT_IDENTIFIER};
-use sp_runtime::traits::{Header as HeaderT, One, Saturating};
+use sp_runtime::traits::{Header as HeaderT, One, Saturating, UniqueSaturatedInto};
 use sp_std::{collections::btree_set::BTreeSet, prelude::*, result};
 
 const MAX_UNCLES: usize = 10;
 
+struct MaxUncleEntryItems<T>(core::marker::PhantomData<T>);
+impl<T> Get<u32> for MaxUncleEntryItems<T>
```

Comment:
I would just do `T: Config` here replacing the where clause, but it's your call.

Reply:
I always use `where` by default, but if the convention in FRAME is to do it inside of the `impl` I'm happy to do so.

Done.

Reply:
Not a convention, simply that I don't find it necessary to write a big where clause for a type parameter that only has 1 trait bound.